meta-psa/recipes-security/psa-storage/psa-storage.bb

```diff
@@ -0,0 +1,38 @@
+# Copyright (c) 2019 Arm Limited and Contributors. All rights reserved.
+#
+# SPDX-License-Identifier: MIT
+
+SUMMARY = "ARM Platform Security Architecture (PSA) Protected Storage Library"
+DEPENDS = ""
+HOMEPAGE = "https://github.com/ARMmbed/psa-storage"
+DESCRIPTION = "A Linux C library reference implementation of the PSA protected_storage.h API"
+
+LICENSE = "Apache-2.0"
+LIC_FILES_CHKSUM = " \
+    file://${WORKDIR}/git/LICENSE;md5=302d50a6369f5f22efdb674db908167a \
+    file://${WORKDIR}/git/apache-2.0.txt;md5=3b83ef96387f14655fc854ddc3c6bd57 \
+    "
+
+SRC_URI = "git://git@github.com/armmbed/psa-storage.git;protocol=ssh;nobranch=1"
+SRCREV = "3572fc55c8cc494bfeb442220f8a91542ed7bb99"
+
+PV = "1.0.0+git${SRCPV}"
```

Comment:
Since we are based on `1.0.0` version this information should be present in the recipe filename (e.g.: psa-storage_1.0.0.bb) and have `PV .= "+git${SRCPV}"`.

Reply:
Done in rework.

Reply:
Fixed in https://github.com/ARMmbed/meta-mbl/pull/619.